sae_lens/training/sae_trainer.py

```diff
@@ -204,23 +212,14 @@ def fit(self) -> TrainingSAE:
         pbar.close()
         return self.sae
 
-    @torch.no_grad()
-    def _estimate_norm_scaling_factor_if_needed(self) -> None:
-        if self.cfg.normalize_activations == "expected_average_only_in":
-            self.activation_store.estimated_norm_scaling_factor = (
-                self.activation_store.estimate_norm_scaling_factor()
-            )
-        else:
-            self.activation_store.estimated_norm_scaling_factor = 1.0
-
```

Comment:
I've moved this into `ActivationsStore` as it seemed more natural.

the general pattern of the change being:

```python
class ParentThing:
    def set_child_property():
        self.thing.property = self.thing.calculate_property()
```
vs
```python
class ChildThing
    def set_property():
        self.property = self.calculate_property()
```